modules/tinymce/src/themes/silver/main/ts/ui/skin/Loader.ts

```
@@ -23,58 +42,99 @@ const loadRawCss = (editor: Editor, key: string, css: string, styleSheetLoader:
-    return loadStylesheet(editor, skinUiCss, editor.ui.styleSheetLoader);
+    const skinUiCssUrl = skinUrl + `/${filenameBase}${suffix}.css`;
+    return { _kind: 'load-stylesheet', url: skinUiCssUrl };
+  });
+};
+
+const loadUiSkins = (editor: Editor, skinUrl: string): Promise<void> => {
+  const loader = editor.ui.styleSheetLoader;
+  const decision = determineCSSDecision(editor, 'skin', skinUrl);
+  switch (decision._kind) {
+    case 'load-raw':
+      const { key, css } = decision;
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)